docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc

```diff
@@ -0,0 +1,89 @@
+This section includes:
+
+. xref:pro-cloning-the-git-repository-{chapter}[Cloning the Git Repository]
+. xref:pro-installing-angular-dependencies-{chapter}[Installing Angular Dependencies]
+. xref:pro-installing-core-dependencies-{chapter}[Installing Core Dependencies]
+. xref:pro-starting-the-core-server-{chapter}[Starting the Core Server]
+. xref:pro-running-the-mobile-and-portal-demo-applications-{chapter}[Running the Mobile and Portal Demo Applications]
+. xref:ref-demo-app-{chapter}[{WFM-RC-NameShort} Demo Application Reference Material]
+
+[id='pro-cloning-the-git-repository-{chapter}']
+= Cloning the Git Repository
+
+.Procedure
```

Comment:
I would say that this is all one procedure as any on there own don't do anything